src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/Main.cs

```diff
@@ -1179,9 +1179,8 @@ public SecondValidator(int _)
         Assert.Equal(DiagDescriptors.ValidatorsNeedSimpleConstructor.Id, diagnostics[0].Id);
     }
 
-    private static bool SupportRemoteExecutionAndNotInBrowser => RemoteExecutor.IsSupported && !PlatformDetection.IsBrowser;
-
-    [ConditionalFact(nameof(SupportRemoteExecutionAndNotInBrowser))]
+    [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+    [SkipOnPlatform(TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "RemoteExecutor is not supported on Browser, iOS, MacCatalyst, or tvOS.")]
```

Comment:
RemoteExecutor.IsSupported is already implemented like this, see: https://github.com/dotnet/arcade/blob/f52865b28bc605ffb724e917f42c886ed7d69350/src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs#L103-L111

We use it all over the place. Something else must be going on. This would definitely not be the only failing test.

Reply:
Thanks! It seems that the recent change https://github.com/dotnet/runtime/pull/88613 has resolved the issue. I will close this PR and the issue once the 24-hour hit count reports no issues.